todo/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from django.core.exceptions import ValidationError
+
+from todo.models import Tag, Task
+
+
+class TagForm(forms.ModelForm):
+    class Meta:
+        model = Tag
+        fields = "__all__"
+
+
+class TaskForm(forms.ModelForm):
+    tags = forms.ModelMultipleChoiceField(
+        queryset=Tag.objects.all(),
+        widget=forms.CheckboxSelectMultiple,
+        required=False
+    )
+
+    class Meta:
+        model = Task
+        fields = "__all__"
```

Comment:
```suggestion
        fields = "__all__"
        
    def clean_deadline(self):
        deadline = self.cleaned_data.get("deadline")
        if deadline and deadline < timezone.now():
            raise ValidationError("Deadline must be in the future")
        return deadline
```
It would be good option to add validation for `deadline` field. At least it can't be in the past from logic point of view.